### PR TITLE
Combine Gallery & Gallery Carousel paragraphs.

### PR DESCRIPTION
**Pull request by Vinay15**

### CODE REVIEW

File: config/install/fvm.settings.yml [hunk 1]
```diff
@@ -38,3 +41,27 @@ block_content__cards: 1
...
+paragraph__view: 0
+paragraph__view__view_modes:
+  preview: 0
+node__gallery: 0
+node__gallery__view_modes:
+  teaser: 0
+paragraph__cards_gallery: 1
+paragraph__cards_gallery__view_modes:
+  gallery: gallery
+  gallery_carousel: gallery_carousel
```
Comment (vaibhavjain-in) [inline config/install/fvm.settings.yml]: The disabled items shall not be part this config. As they may not be enabled on new site install.

File: modules/ezcontent_paragraphs/ezcontent_paragraphs.routing.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+ezcontent_paragraphs.display:
+  path: '/paragraph-view/{paragraph}/{view_mode}'
```
Comment (vaibhavjain-in) [inline modules/ezcontent_paragraphs/ezcontent_paragraphs.routing.yml:2]: `paragraph` is paragraphs ID ?

Comment (Vinay15) [reply modules/ezcontent_paragraphs/ezcontent_paragraphs.routing.yml:2]: yes.

**Review by vaibhavjain-in — APPROVED** [review]
LGTM

**Pull request merged**